### DISCUSSION TITLE
支持数据科学网站开发工具 shinyproxy 部署

### DISCUSSION THREAD

**Discussion by fei0810** [zeabur/.github#31] [General]
希望增加数据科学R shiny 相关网站部署工具模板 https://www.shinyproxy.io/ 
可以参考railwayhttps://github.com/railwayapp-templates/shiny/

**Comment by yuaanlin** [comment]
感谢建议，我们诚挚邀请您到 Discord 交流群与我们互动 https://zeabur.com/dc
